### PR TITLE
Runtimecfg

### PR DESCRIPTION
**Pull request by bcrochet**
**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: pkg/operator/bootstrap.go [hunk 1]
```diff
@@ -145,7 +150,39 @@ func RenderBootstrap(
 	}, {
 		name:     "manifests/machineconfigserver/kube-apiserver-serving-ca-configmap.yaml",
 		filename: "manifests/kube-apiserver-serving-ca-configmap.yaml",
+	}, {
```
Comment (iamemilio) [inline pkg/operator/bootstrap.go:153]: I think you forgot to remove these

Comment (celebdor) [reply pkg/operator/bootstrap.go:153]: definitely

File: pkg/operator/bootstrap.go [hunk 2]
```diff
@@ -145,7 +150,39 @@ func RenderBootstrap(
...
+		name:     "manifests/baremetal/keepalived.conf.tmpl",
+		filename: "static-pod-resources/keepalived/keepalived.conf.tmpl",
 	}}
+	baremetalManifests := []struct {
+		name     string
+		data     []byte
+		filename string
+	}{{
+		name:     "manifests/baremetal/coredns.yaml",
+		filename: "manifests/coredns.yaml",
```
Comment (mandre) [inline pkg/operator/bootstrap.go:172]: This should be moved to `baremetal/manifests/coredns.yaml` to match https://github.com/celebdor/installer/commit/134d74cdab091303721b00d2111330c862420b23

File: pkg/operator/bootstrap.go [hunk 3]
```diff
@@ -145,7 +150,39 @@ func RenderBootstrap(
...
+	}, {
+		name:     "manifests/baremetal/coredns-corefile.tmpl",
+		filename: "static-pod-resources/coredns/Corefile.tmpl",
```
Comment (mandre) [inline pkg/operator/bootstrap.go:175]: `baremetal/static-pod-resources/coredns/Corefile.tmpl`

File: pkg/operator/bootstrap.go [hunk 4]
```diff
@@ -145,7 +150,39 @@ func RenderBootstrap(
...
+	}, {
+		name:     "manifests/baremetal/keepalived.yaml",
+		filename: "manifests/keepalived.yaml",
```
Comment (mandre) [inline pkg/operator/bootstrap.go:178]: `baremetal/manifests/keepalived.yaml`

File: pkg/operator/bootstrap.go [hunk 5]
```diff
@@ -145,7 +150,39 @@ func RenderBootstrap(
...
+	}, {
+		name:     "manifests/baremetal/keepalived.conf.tmpl",
+		filename: "static-pod-resources/keepalived/keepalived.conf.tmpl",
```
Comment (mandre) [inline pkg/operator/bootstrap.go:181]: `baremetal/static-pod-resources/keepalived/keepalived.conf.tmpl`

File: templates/worker/00-worker/baremetal/files/dhcp-dhclient-conf.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/dhcp/dhclient.conf.template"
```
Comment (mandre) [inline templates/worker/00-worker/baremetal/files/dhcp-dhclient-conf.yaml]: This should be `/etc/dhcp/dhclient.conf`

File: templates/worker/00-worker/baremetal/files/dhcp-dhclient-conf.yaml [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+contents:
+  inline: |
+    supersede domain-search "{{ .EtcdDiscoveryDomain }}";
+    prepend domain-name-servers {{`{{ .NonVirtualIP }}`}};
```
Comment (mandre) [inline templates/worker/00-worker/baremetal/files/dhcp-dhclient-conf.yaml]: also, NonVirtualIP is never defined it seems, and it looks like we're not making use of the `/usr/local/bin/non_virtual_ip` script above. Are we missing a patch?

**Pull request closed without merging**